ui/src/app/(index)/_components/AfterLogin.tsx

```diff
@@ -0,0 +1,99 @@
+import User from '@/access/User';
+import Role from '@/access/Role';
+import Image from 'next/image';
+import {KeyRound} from 'lucide-react';
+import UserInfoItem from '@/app/(index)/_components/UserInfoItem';
+
+const AfterLogin = (
+    {
+        currentUser,
+        numberOfGroupings,
+        numberOfMemberships
+    }: {
+        currentUser: User,
+        numberOfGroupings: number,
+        numberOfMemberships: number
+    }) => {
```

Comment:
Could you format this part like this: (move the curly-braces over)
```
const AfterLogin = ({
    currentUser,
    numberOfGroupings,
    numberOFMemberhsips
}: {
    currentUser: User,
    numberOfGroupings: number,
    numberOfMemberships: number
}) => {
```
Thanks!

There are a few other places in this PR like that too, I will look into a code formatter so that it will automatically format it for you in the future.